vendor/modules.txt

```diff
@@ -7,7 +7,7 @@ github.com/BurntSushi/toml
 github.com/beorn7/perks/quantile
 # github.com/cespare/xxhash/v2 v2.1.1
 github.com/cespare/xxhash/v2
-# github.com/cloudevents/sdk-go/protocol/amqp/v2 v2.3.1-0.20210302080936-5c462007a5d5
+# github.com/cloudevents/sdk-go/protocol/amqp/v2 v2.4.1
```

Comment:
what was introduced or fixed in v2.4.1 ?

Reply:
not sure. It is generated by `go mod tidy`.